src/styles/globals.css

```diff
@@ -27,5 +28,5 @@ a {
 
 .nav-tabs .nav-item.show .nav-link,
 .nav-tabs .nav-link.active {
-  background-color: #00ACED !important;
+  background-color: #95daf4 !important;
```

Comment:
@Pseudoman21 please see images. ang tiga mean kong default is yung wala ka ng gagalawin sa CSS. chineck mo ba reference? anyway inadd ko na lang dito image

![image](https://user-images.githubusercontent.com/28330063/233027170-bc5c798f-e3f2-4bdb-b995-3a4dda5fa8f2.png)

![image](https://user-images.githubusercontent.com/28330063/233027236-7b8a0f72-dd46-455e-ae01-562d175c9f95.png)

Reply:
> @Pseudoman21 please see images. ang tiga mean kong default is yung wala ka ng gagalawin sa CSS. chineck mo ba reference? anyway inadd ko na lang dito image
> 
> ![image](https://user-images.githubusercontent.com/28330063/233027170-bc5c798f-e3f2-4bdb-b995-3a4dda5fa8f2.png)
> 
> ![image](https://user-images.githubusercontent.com/28330063/233027236-7b8a0f72-dd46-455e-ae01-562d175c9f95.png)

![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/28330063/233027947-52146428-f02c-4921-b777-69e5b6f524c2.png)

Reply:
Opo Kuya, removed na po ini did po sa dae pa nakasync na changes. Si Chan po nag add ki background image diyan, bale tigpalitan ko nalang po kuta ki lighter color. Pero didi po sa isisync palang removed na po siya Kuya hehe